### DISCUSSION TITLE
SP Racing F3 Deluxe

### DISCUSSION THREAD

**Discussion by CWTechniques** [iNavFlight/inav#6668] [Q&A] [answered]
I expect this is an old topic as the board is pretty old but I searched and couldn't find any previous responses. I have an SP Racing F3 deluxe board I was hoping to use with iNav for an odd 6 inch frame I have and as both are a bit odd now I was hoping to put them together in a build. The issue I have is that while I was able to flash iNav 2.6.1 and 2.5.0 to the board when I connect to it the CPU usage spikes to 800-900% and the board is more or less locked up. If I flash Betaflight 4.0.6 it's fine at 25%, I haven't tried Cleanflight yet but really I wanted to use the iNav GPS functions. I realize that 2.6.1 is the last release the F3 boards are supported in as they are deprecated but I was hoping for some functionality. Truth be told the board is likely a clone but as it seems to work ok for fairly current Betaflight I don't think that is the issue. I'm hoping that someone can offer a remedy for the situation or some better insight into which version of iNav will work for this board so I don't have to go through the fun of downloading and flashing configurators and releases until I find it myself. 

Thanks in advance

CWTechniques

**Comment by DzikuVx** [comment, marked as answer]
F3 boards are end of live and end of support. Try flashing older releases like INAV 2.4 or 2.3

**Reply by CWTechniques** [reply]
I tried INAV 2.4 and 2.5 and both exhibit the same issues I have been having but Cleanflight 2.5 and Betaflight 4.0.6 firmwares seem to work fine. I am not sure if there is something I am missing or I just haven't found the correct version yet. I have tried firmwares 2.6, 2.5, 2.4 and 2.3 and all have the same result. I'm not sure what is going on at this point.

**Comment by hobsicko** [comment]
tray INAV-Configurator_win64_2.6.1 inav 2.6.1
![inav2 6 1](https://user-images.githubusercontent.com/93543881/157736850-c5280a3a-63f4-401c-b1ea-7b04cd434d18.png)
1
